sample-apps/discount-functions-sample-app/extensions/fixed-product-discount/src/api.rs

```diff
@@ -0,0 +1,123 @@
+#![allow(dead_code)]
+
+pub type Boolean = bool;
+pub type Float = f64;
+pub type Int = i64;
```

Comment:
[Integers in GraphQL are i32](https://spec.graphql.org/October2021/#sel-GAHXRDABABsCuvW):

> The Int scalar type represents a signed 32-bit numeric non-fractional value.